FpsCheck.py

```diff
@@ -243,7 +243,9 @@ def is_eTLD_Plus1(self, site):
         Returns:
             boolean with truth value dependent on value of get_public_suffix
         """
-        return self.etlds.get_public_suffix(site, strict=True) is not None
+        ps = self.etlds.get_public_suffix(site, strict=True)
+        return (ps is not None and ps == site)
```

Comment:
```suggestion
        assert site is not None
        return self.etlds.get_public_suffix(site, strict=True) == site
```
I think this is equivalent and simpler? We should never be passing in `None` as the site, right?

Reply:
I was wrongly worrying that if ps was `None` that it would throw an error, forgetting that `None == value` is totally valid Python. I'll add this change.